src/RedemptionPool.sol

```diff
@@ -98,74 +128,88 @@ contract RedemptionPool is Ownable {
 
     /// @notice deposit GRO tokens to the pool before the deadline
     /// @param _amount amount of GRO tokens to deposit
-    function deposit(uint256 _amount) external {
-        // Checks that the DEADLINE has not passed
-        if (block.timestamp > DEADLINE) {
-            revert RedemptionErrors.DeadlineExceeded();
-        }
+    function deposit(uint256 _amount) external onlyBeforeDeadline {
+        if (_amount <= 0) revert RedemptionErrors.GreaterThanZeroOnly();
```

Comment:
Cannot be negative

Reply:
And even if zero then still is no problem so can just remove this check?